CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 ## Unreleased
 
+### Fixed
+
+- Fix ECR regex separating tag on forward slash and period @KyleGoyette https://github.com/wandb/wandb/pull/7837
```

Comment:
maybe also mention that it is related to launch, otherwise it's probably only clear to users that know about launch